2.7.0/final.md

```diff
@@ -0,0 +1,1045 @@
+# PyTorch 2.7.0 Release Notes
+- [Highlights](#highlights)
+- [Backwards Incompatible Changes](#backwards-incompatible-changes)
+- [Deprecations](#deprecations)
+- [New Features](#new-features)
+- [Improvements](#improvements)
+- [Bug fixes](#bug-fixes)
+- [Performance](#performance)
+- [Documentation](#documentation)
+- [Developers](#developers)
+- [Security](#security)
+
+# Highlights
+TODO @seemethere
```

Comment:
Perhaps add a highlight for Blackwell:

"Blackwell support added across native kernels, CUDA math libraries, and `torch.compile`?"

Reply:
Should this also be under New features?

Reply:
Added under new features